config/routes.js

```diff
@@ -28,17 +28,8 @@ router.get('/search',
   }
 );
 
-// Only get open things for display
-router.get('/results-open',
-  validateLocation,
-  urlUtils.urlForPharmacyPostcodeSearch,
-  servicesMiddleware.getPharmacies,
-  servicesMiddleware.getPharmacyOpeningTimes,
-  servicesMiddleware.prepareOpenThingsForRender,
-  servicesMiddleware.renderServiceResults
-);
-
 router.get('/results',
+  validateLocation,
```

Comment:
I'm still able to navigate to http://localhost:3000/results-open by virtue of the `route.get('/:view',` line in config/routes. results-open.nunjuck needs to be deleted.